use-cases/Create_Data_Quality_Monitors_Use_Case.ipynb

```diff
@@ -39,7 +40,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": null,
+      "execution_count": 47,
```

Comment:
maybe clear out this state. NBD

Reply:
oh my B i was trigger happy on the merge